libs/sdk-ui-tests-e2e/cypress/tools/widgetConfiguration.ts

```diff
@@ -184,6 +191,11 @@ export class WidgetConfiguration {
         this.getElement().find(".gd-input-field").clear().type(text);
         return this;
     }
+
+    checkFilterCheckbox(text: string) {
```

Comment:
param should be name as attributeFilter, or item

Reply:
I have rename it to _selectFilterCheckbox()_ . It more clearly about the action and the object that the function performs.